README.md

```diff
@@ -59,6 +59,7 @@ The following annotations are supported:
 ||`ingress.kubernetes.io/secure-backends`|[true\|false]|-|
 ||`ingress.kubernetes.io/secure-verify-ca-secret`|secret name|-|
 ||[`ingress.kubernetes.io/session-cookie-name`](#affinity)|cookie name|-|
+|`[0]`|[`ingress.kubernetes.io/session-cookie-strategy`](#affinity)|cookie strategy|-|
```

Comment:
This PR belongs to v0.6, so use `[1]` instead and create also a `* ``[1]`` only in ``snapshot`` tag` entry just before this table.

Change also `cookie strategy` text to a list of options. Have a look at eg `forwardfor` configmap option.